src/services/wizdom.spfx.service.ts

```diff
@@ -41,10 +44,13 @@ export class WizdomSpfxServices {
                     console.error("Invalid developermode", ex);                    
                 }
             }
+
+            let aadHttpClientPromise: Promise<AadHttpClient> = this.spContext.aadHttpClientFactory.getClient("402cbaeb-c52a-43b6-b886-4ad1c44cab6a");
```

Comment:
Not entirely sure, how promise works in this case. Will this start the creation of the client, also for non-saas installationen. If thats the case, maybe change this to a factory thingy

Reply:
It will start the creation of the client, also for non-SaaS in parallel and then await it when needed, the hope is that it will be done initializing when it's needed.

Since it's not awaited in any of the non-SaaS codepaths I don't expect it to have much impact for that, and I believe that was a trade-off I just kinda accepted to be able to initialize it before the call to tennant properties and having to wait for those to resolve first to see if it's SaaS (potentially 3 http requests)